rasa/nlu/classifiers/sklearn_intent_classifier.py

```diff
@@ -100,7 +100,7 @@ def train(
                 "Can not train an intent classifier as there are not "
                 "enough intents. Need at least 2 different intents. "
                 "Skipping training of intent classifier.",
-                docs=DOCS_URL_TRAINING_DATA_NLU,
+                docs=DOCS_URL_TRAINING_DATA,
```

Comment:
What about still linking to the `NLU Training Data` section?

Reply:
:+1: